nginx.conf

```diff
@@ -16,8 +16,47 @@ server {
   add_header Content-Security-Policy-Report-Only "upgrade-insecure-requests; default-src 'none'; script-src 'self' 'unsafe-inline' cdn.ravenjs.com www.googletagmanager.com www.google-analytics.com js.hs-scripts.com js.hs-analytics.net; connect-src 'self' sentry.io reload.getsentry.net api.amplitude.com; img-src 'self' www.google-analytics.com www.facebook.com track.hubspot.com saucelabs.com; style-src 'self' 'unsafe-inline' fonts.googleapis.com; font-src 'self' fonts.gstatic.com; report-uri https://sentry.io/api/1297620/security/?sentry_key=b3cfba5788cb4c138f855c8120f70eab" always;
 
   rewrite ^/(?:hosted|on-premise)/(.*)$ /$1$is_args$args redirect;
-  rewrite ^/learn/(data-forwarding|issue-owners|membership|notifications|pricing|sso|search|quotas)/$ /product/$1$is_args$args redirect;
-
+  
+  rewrite ^/clientdev/(.*)$ /development/sdk-dev/$1$is_args$args redirect;
+  rewrite ^/integrations/(.*)$ /workflow/integrations/$1$is_args$args redirect;
+  rewrite ^/internal/(api|docs|contributing|environment)/$ /development/contribute/$1$is_args$args redirect;
```

Comment:
Is there a reason you opted to be specific here when it _seems_ like you don't need to be?